.pre-commit-config.yaml

```diff
@@ -27,7 +27,7 @@ repos:
     hooks:
       - id: blacken-docs
   - repo: https://github.com/PyCQA/flake8
-    rev: 4.0.1
+    rev: 7.0.0
```

Comment:
Not fully relevant here but worth updating checkov as well. Anticipate that will cause some checks to fail